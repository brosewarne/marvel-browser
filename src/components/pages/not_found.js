import React from 'react';
import Typography from 'material-ui/Typography';
import Paper from 'material-ui/Paper';

import './pages.css';

/**
 * Render the 404 Page
 * @returns {Node} - The rendered 404 Page
 * @constructor
 */
export const NotFound = () => {
    return (
        <div>
            <content className="pageContainer">
                <Paper elevation={12}>
                    <Typography color="primary" className="appHeader" align="center" >
                        <h1 className="itemProfileTitle">Sorry we couldn;t find the page you're looking for :(</h1>
                    </Typography>
                </Paper>
            </content>
        </div>
    );
};
