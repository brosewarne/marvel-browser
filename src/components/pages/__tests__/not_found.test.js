import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import { NotFound } from '../not_found';

configure({ adapter: new Adapter() });

describe('<NotFound />', () => {
    it('renders correctly', () => {
        const wrapper = shallow(
            <NotFound />
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
