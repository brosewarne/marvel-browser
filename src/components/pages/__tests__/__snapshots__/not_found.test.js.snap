// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders correctly 1`] = `
<div>
  <content
    className="pageContainer"
  >
    <WithStyles(Paper)
      elevation={12}
    >
      <WithStyles(Typography)
        align="center"
        className="appHeader"
        color="primary"
      >
        <h1
          className="itemProfileTitle"
        >
          Sorry we couldn;t find the page you're looking for :(
        </h1>
      </WithStyles(Typography)>
    </WithStyles(Paper)>
  </content>
</div>
`;
